lib/cfnvpn/templates/vpn.rb

```diff
@@ -164,12 +173,23 @@ def render(name, config)
 
         if cidr_routes.any?
           cidr_routes.each do |route|
-            EC2_ClientVpnRoute(:"#{route[:cidr].resource_safe}VpnRoute") {
-              Description "cfnvpn static route for #{route[:cidr]}. #{route[:desc]}".strip
-              ClientVpnEndpointId Ref(:ClientVpnEndpoint)
-              DestinationCidrBlock route[:cidr]
-              TargetVpcSubnetId route[:subnet]
-            }
+            # to aide in the migration from single to HA routes if the vpn is HA
+            if route[:subnets]
+              target_subnets = route[:subnets]
```

Comment:
This comes across a little confusing with the naming 'target_subnets', at first I thought it was a duplication of the changes above but it appears to be different functionality. Maybe worth changing the naming or am I being pedantic?

Reply:
it's the same thing but implemented in a different way. 
- `dns_routes` are auto generated by the lambda route populator
- `cidr_routes` are static routes which can be created through cloudformation

Client vpn routes are created by routing a cidr to what AWS refer to as a "Target Subnet" which is just a  subnet in a vpc

```rb
EC2_ClientVpnRoute(...) {
  DestinationCidrBlock ...
  TargetVpcSubnetId ...
}
```